File: spec-api-python/raw.py
"""
Get data
"""
import seaborn as sns

# + tags=["parameters"]
upstream = None
product = None
# -

# +
df = sns.load_dataset('iris')
df.columns = [
    'sepal_length', 'sepal_width', 'petal_length', 'petal_width', 'class'
]
df.head()
# -

# +
df.to_csv(str(product['data']), index=False)
# -

File: spec-api-r/raw.R
# + tags=["parameters"]
upstream = NULL
product = NULL
# -


df = read.csv('https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data',
header=FALSE)
colnames(df) = list('sepal_length', 'sepal_width','petal_length', 'petal_width', 'class')

write.csv(df, product$data)